exporter/exporter.go

```diff
@@ -276,40 +276,210 @@ func GenerateCat1(patient []byte, measures []byte, valueSets []byte, startDate i
+					TagName: "representedOrganization",
+				},
+			},
+		},
+		Custodian: models.Author{
+			Entity: models.Entity{
+				Ids: []models.CDAIdentifier{
+					models.CDAIdentifier{
+						Root:      "custodianRoot",
+						Extension: "custodianExtension",
+					},
+				},
+			},
+			Person: models.Person{
+				First: "",
+				Last:  "",
+			},
+			Organization: models.Organization{
+				Entity: models.Entity{
+					Ids: []models.CDAIdentifier{
+						models.CDAIdentifier{
+							Root:      "custodianOrganizationRoot",
+							Extension: "custodianOrganzationExt",
+						},
+					},
+					Addresses: []models.Address{
+						models.Address{
+							Street: []string{
+								"202 Burlington Road",
+								"Apartment 1",
+							},
+							City:    "Bedford",
+							State:   "MA",
+							Zip:     "01730",
+							Country: "USA",
+							Use:     "PUB",
+						},
+					},
+					Telecoms: []models.Telecom{
+						models.Telecom{
+							Use:   "WP",
+							Value: "1(781)2712000",
+						},
+					},
+				},
+				Name:    "CustodianOrganization",
+				TagName: "representedCustodianOrganization",
+			},
+		},
+		Authenticator: models.Authenticator{
+			Author: models.Author{
+				Entity: models.Entity{
+					Ids: []models.CDAIdentifier{
+						models.CDAIdentifier{
+							Root:      "legalAuthenticatorRoot",
+							Extension: "legalAuthenticatorExt",
+						},
+					},
+					Addresses: []models.Address{
+						models.Address{
+							Street: []string{
+								"202 Burlington Road",
+								"Apartment 1",
+							},
+							City:    "Bedford",
+							State:   "MA",
+							Zip:     "01730",
+							Country: "USA",
+							Use:     "PUB",
+						},
+					},
+					Telecoms: []models.Telecom{
+						models.Telecom{
+							Use:   "WP",
+							Value: "1(781)2712000",
+						},
+					},
+				},
+				Time: atime2,
+				Person: models.Person{
+					First: "Legal",
+					Last:  "Authenticator",
+				},
+				Organization: models.Organization{
+					Entity: models.Entity{
+						Ids: []models.CDAIdentifier{
+							models.CDAIdentifier{
+								Root:      "legalAuthenticatorOrgRoot",
+								Extension: "legalAuthenticatorOrgExt",
+							},
+						},
+					},
+					Name:    "LegalAuthenticatorOrg",
+					TagName: "representedOrganization",
+				},
+			},
+		},
+	}
+
+	aggCount := doc.AggregateCount{
+		Populations: 	  	mr.Populations,
+		PopulationGroups: 	mr.PopulationGroups,
+	}
+	aggCounts := make(map[string]doc.AggregateCount, 1)
+	aggCounts[m.HQMFID] = aggCount
+	log.Println(aggCounts)
+	ms := &doc.MeasureSection{
+		Measure: models.Measure{
+			ID:        "measure test id",
+			HQMFID:    m.HQMFID,
+			Name:      m.Name,
+			HQMFSetID: m.HQMFSetID,
+		},
+		Results: aggCounts,
+	}
 
 	if version == "" {
 		version = "r2"
 	}
 
-	data, err := AssetDir("templates/cat3/" + version)
-	if err != nil {
-		fmt.Println(err)
-	}
+	//data, err := AssetDir("templates/cat3/" + version)
+	//if err != nil {
+	//	fmt.Println(err)
+	//}
```

Comment:
For all of these commented sections, would it make more sense to simply delete them or are we not done with that dummy code yet?